app/script/event/EventRepository.coffee

```diff
@@ -355,7 +357,8 @@ class z.event.EventRepository
         return @cryptography_repository.decrypt_event event
         .catch (decrypt_error) =>
           # Get error information
-          error_code = decrypt_error.code or 999
+          decrypt_error.code = UNKNOWN_DECRYPT_ERROR_CODE if not decrypt_error.code
```

Comment:
Why? Not setting the 999 on the decrypt_error.code but only on the message was on purpose. Otherwise we will mask all unknown error messages in report_decrypt_error from Raygun and Localytics.

``error_code = decrypt_error.code or UNKNOWN_DECRYPT_ERROR_CODE``